templates/zerver/help/only-allow-admins-to-invite-new-users.md

```diff
@@ -0,0 +1,13 @@
+# Only allow admins to invite new users.
+By enabling this will only allow administrators to invite new users within
+the organisation. This can only be configured from an Administrator's account.
+
+1. Locate the cog
+(![cog](/static/images/help/cog.png)) on the upper right corner of the right
+sidebar and click on it to relase the dropdown menu. Select the **Administration**
+option. This will open the Administration page of your organization.
+![Administration](/static/images/help/administration.png)
+2. Under Organization settings select the **E-mail invitation required** and
+**Only admins may invite** checkboxes.
+![Organization settings](/static/images/help/settings-administration.png)
+3. Click on the blue **Save changes**.
```

Comment:
I would add "button" to the end of that sentence if I were you. So, it would be "3. Click on the blue **Save changes** button".